packaging/docker_sys_config.sh

```diff
@@ -1,12 +1,25 @@
 install -o root -g root -d /etc/sysconfig/network-scripts
-echo -e "[logging]\nlevel=TRACE\ndomains=ALL\n" > \
-    /etc/NetworkManager/conf.d/97-docker-build.conf
-echo -e "[device]\nmatch-device=*\nmanaged=0\n" >> \
-    /etc/NetworkManager/conf.d/97-docker-build.conf
-echo -e "[main]\nno-auto-default=*\n" >> \
-    /etc/NetworkManager/conf.d/97-docker-build.conf
-sed -i 's/#RateLimitInterval=30s/RateLimitInterval=0/' \
-    /etc/systemd/journald.conf
-sed -i 's/#RateLimitBurst=1000/RateLimitBurst=0/' \
+
+cat > /etc/NetworkManager/conf.d/97-trace-logging.conf <<EOF
+[logging]
+level=TRACE
+domains=ALL
+EOF
+
+sed -i -e 's/^#RateLimitInterval=/RateLimitInterval=0/' \
```

Comment:
What happened to the existing values? You do not need to replace them as well?

nit: Maybe add an `\` after `sed -i`.

Reply:
What do you mean with existing values? The file conains only commented values and the previous script did also only change the commented lines.

I do not believe atm that the backslash is needed there.

Reply:
> I do not believe atm that the backslash is needed there.

You are just splitting in an unexpected place. Two `-e`, deserves two lines IMO.
But again, if you do not like it, it is not a must.

The comment has a value, like: `#RateLimitInterval=30`. If youdo not include the `30`, then it will not be replaced/removed. Or do I miss something here?

Reply:
It just matches the line prefix since there is no dollar sign to match the end

Reply:
Yes, you are right. I confused this with how the ansible lineinfile module works. Will fix this by adding `.*` after the equal sign.